api/src/test/scala/uk/ac/wellcome/platform/api/ApiWorksTest.scala

```diff
@@ -0,0 +1,123 @@
+        "es.xpack.enabled" -> "true",
+        "es.xpack.user" -> "elastic:changeme",
+        "es.xpack.sslEnabled" -> "false",
+        "es.sniff" -> "false"
+      )
+    )
+  val itemType = "item"
+
+  test("it should return a list of works") {
+
+    val firstIdentifiedUnifiedItem =
+      identifiedUnifiedItemWith(canonicalId = "1234",
+                                label = "this is the first image title")
+    val secondIdentifiedUnifiedItem =
+      identifiedUnifiedItemWith(canonicalId = "4321",
+                                label = "this is the second image title")
+    val thirdIdentifiedUnifiedItem =
+      identifiedUnifiedItemWith(canonicalId = "9876",
+                                label = "this is the third image title")
+
+    insertIntoElasticSearch(firstIdentifiedUnifiedItem)
+    insertIntoElasticSearch(secondIdentifiedUnifiedItem)
+    insertIntoElasticSearch(thirdIdentifiedUnifiedItem)
+
+    eventually {
+      server.httpGet(
+        path = "/catalogue/v0/works",
+        andExpect = Status.Ok,
+        withJsonBody = s"""
+            |{
+            |  "@context": "http://${server.externalHttpHostAndPort}/catalogue/v0/context.json",
+            |  "type": "ResultList",
+            |  "pageSize": 10,
+            |  "results": [
+            |   {
+            |     "type": "Work",
+            |     "id": "${firstIdentifiedUnifiedItem.canonicalId}",
+            |     "label": "${firstIdentifiedUnifiedItem.unifiedItem.label}"
+            |   },
+            |   {
+            |     "type": "Work",
+            |     "id": "${secondIdentifiedUnifiedItem.canonicalId}",
+            |     "label": "${secondIdentifiedUnifiedItem.unifiedItem.label}"
+            |   },
+            |   {
+            |     "type": "Work",
+            |     "id": "${thirdIdentifiedUnifiedItem.canonicalId}",
+            |     "label": "${thirdIdentifiedUnifiedItem.unifiedItem.label}"
+            |   }
+            |  ]
+            |}
+          """.stripMargin
+      )
+    }
+  }
+
+  test("it should return a single work when requested with id") {
+    val identifiedUnifiedItem =
+      identifiedUnifiedItemWith(canonicalId = "1234",
+                                label = "this is the first image title")
+    insertIntoElasticSearch(identifiedUnifiedItem)
+
+    eventually {
+      server.httpGet(
+        path = "/catalogue/v0/works/1234",
+        andExpect = Status.Ok,
+        withJsonBody = s"""
+            |{
+            | "@context": "http://${server.externalHttpHostAndPort}/catalogue/v0/context.json",
+            | "type": "Work",
+            | "id": "1234",
+            | "label": "this is the first image title"
+            |}
+          """.stripMargin
+      )
+    }
+  }
+
+  test("it should return a not found error when requesting a single work with a non existing id") {
+    server.httpGet(
+      path = "/catalogue/v0/works/non-existing-id",
+      andExpect = Status.NotFound,
+      withJsonBody = ""
```

Comment:
Yes we should, I'm still definitely what it should look like. Will add a separate ticket.